src/main/java/org/isf/disease/gui/DiseaseEdit.java

```diff
@@ -302,14 +302,25 @@ public void actionPerformed(java.awt.event.ActionEvent e) {
 					disease.setIpdOutInclude(includeIpdOutCheckBox.isSelected());
 					
 					boolean result = false;
+					Disease savedDisease = null;
 					try{
 						if (insert) { // inserting
-							result = manager.newDisease(disease);
+							savedDisease = manager.newDisease(disease);
+							if(savedDisease != null) {
```

Comment:
This is just a code style comment; the project uses `if (` instead of `if(` [a space between `if` and `(`] as you have added in this PR.  Thanks.

Reply:
@dbmalkovsky I have fixed the code style issue